### PR TITLE
Converted the plugin to now work via Composer

### PR DESCRIPTION
**Pull request by jcomack**
Converted the plugin to now work via Composer and moved the main class to the classes folder.
Also added the old autoloading method for the sake of backwards compatibility.

### CODE REVIEW

**Comment by atimmer** [comment]
Code looks good, but the build should pass.

File: wpseo-news.php [hunk 1]
```diff
@@ -30,349 +30,15 @@
...
-			return wpseo_xml_sitemaps_base_url( $sitemap_name . '-sitemap.xml' );
-		}
-
-		return $sitemap_name;
+if ( file_exists( dirname( __FILE__ ) . '/vendor/autoload_52.php' ) ) {
+	require dirname( __FILE__ ) . '/vendor/autoload_52.php';
+}
 
-	}
+if ( ! file_exists( dirname( __FILE__ ) . '/vendor/autoload_52.php' ) ) {
```
Comment (omarreiss) [inline wpseo-news.php]: Please remove these lines and remove the legacy autoloader. If we let composer do the autoloading, then we let composer do the autoloading. That was one of the main reasons for composerifying anyway.

**Comment by omarreiss** [comment]
@jcomack CR done. One issue found.

**Comment by andizer** [comment]
@omarreiss There is something wrong with the beta script, it won't call composer install.

**Comment by andizer** [comment]
@omarreiss  I've just fixed it in the beta script.

**Comment by andizer** [comment]
Acceptance done :+1:

**Pull request merged**